decidim-core/lib/decidim/resource_manifest.rb

```diff
@@ -36,6 +36,9 @@ class ResourceManifest
     # The main card to render an instance of the resource.
     attribute :card, String
 
+    # The reported content to render an instance of the resource.
+    attribute :reported_content, String
```

Comment:
maybe we can rename it to
```suggestion
    attribute :reported_content_cell, String
```
?